pkg/vault/test/foundry/BatchRouterERC4626Pool.t.sol

```diff
@@ -155,16 +158,16 @@ contract BatchRouterERC4626PoolTest is BaseERC4626BufferTest {
             );
         }
         {
-            (, , , uint256[] memory lastBalancesLiveScaled18) = vault.getPoolTokenInfo(address(erc4626Pool));
+            (, , uint256[] memory balances, ) = vault.getPoolTokenInfo(address(erc4626Pool));
```

Comment:
In theory, lastBalancesLive should be equal to the underlying amount, but it's not. When we call "convertToShares", we multiply the amount by the totalSupply and divide by the assets allocated in the wrapper directly (in most cases), which does not introduce rounding issues.

However, lastBalancesLive is the `wrappedAmount` multiplied by convertToAssets(FixedPoint.ONE), which is the rate of one share in a 18 decimal number. This introduces a rounding imprecision proportional to the `wrappedAmount`.